### DISCUSSION TITLE
Is there a way we can add a mention of the refetchType option to the invalidateQueries documentation?

### DISCUSSION THREAD

**Discussion by dqhendricks** [TanStack/query#7601] [General] [answered]
Google took me to the guide instead of the reference, which is why I couldn't find it. Sorry for the trouble!

**Comment by TkDodo** [comment, marked as answer]
`refetchType` is documented right here: https://tanstack.com/query/latest/docs/reference/QueryClient/#queryclientinvalidatequeries
